static/app/gettingStartedDocs/dotnet/maui.tsx

```diff
@@ -204,10 +209,26 @@ const onboarding: OnboardingConfig = {
   ],
 };
 
+const crashReportOnboarding: OnboardingConfig = {
+  introduction: () => getCrashReportModalIntroduction(),
+  install: (params: Params) => getCrashReportGenericInstallStep(params),
```

Comment:
just noticed, nit for sure, but this type of thing can be written as:
```suggestion
  introduction: getCrashReportModalIntroduction,
  install: getCrashReportGenericInstallStep,
```
as long as the props are the same (same types obvs, but also same number of props).

ie `(params: Params)`  matches with `(params)` on like 214. 

Places to be careful of this are like `[].forEach` and `[].map` because they pass 3 params into the callback, so if your callback is expecting an optional 2nd param it'll get something unexpected! ie: `[1, 2, 3].forEach(console.log)` is will print more than `[1,2,3].forEach(num => console.log(num))`

Reply:
i think for the sake of having to change 60 more files i won't apply this suggestion for these PRs but will definitely keep this in mind for future PRs🫡

Reply:
yes, def for other times, not now.